vaadin-testbench-unit/src/main/java/com/vaadin/flow/component/listbox/MultiSelectListBoxWrap.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (C) 2022 Vaadin Ltd
+ *
+ * This program is available under Commercial Vaadin Developer License
+ * 4.0 (CVDLv4).
+ *
+ *
+ * For the full License, see <https://vaadin.com/license/cvdl-4.0>.
+ */
+package com.vaadin.flow.component.listbox;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import com.vaadin.flow.component.ItemLabelGenerator;
+import com.vaadin.testbench.unit.ComponentWrap;
+import com.vaadin.testbench.unit.Wraps;
+
+/**
+ * Test wrapper for MultiSelectListBox components.
+ *
+ * @param <T>
+ *            component type
+ * @param <V>
+ *            value type
+ */
+@Wraps(fqn = { "com.vaadin.flow.component.listbox.MultiSelectListBox" })
+public class MultiSelectListBoxWrap<T extends MultiSelectListBox<V>, V>
+        extends ComponentWrap<T> {
+    /**
+     * Wrap given component for testing.
+     *
+     * @param component
+     *            target component
+     */
+    public MultiSelectListBoxWrap(T component) {
+        super(component);
+    }
+
+    /**
+     * Get the currently selected item.
+     *
+     * @return current selection
+     */
+    public Set<V> getSelected() {
+        return getComponent().getValue();
+    }
+
+    /**
+     * Select item(s) by client string representation.
+     *
+     * @param selection
+     *            item representation string
+     */
+    public void selectItems(String... selection) {
+        ensureComponentIsUsable();
+        if (selection == null) {
+            getComponent().setValue(null);
+            return;
+        }
+        List<String> items = Arrays.asList(selection);
+        final List<V> filtered = getSuggestionItems().stream()
+                .filter(item -> items.contains(getItemLabel(item)))
+                .collect(Collectors.toList());
+        if (filtered.size() != items.size()) {
+            throw new IllegalArgumentException(
+                    "Selection contained items that didn't have a selection");
+        }
+        getComponent().select(filtered);
+    }
+
+    /**
+     * Deselect item(s) by client string representation.
+     *
+     * @param selection
+     *            item representation string
+     * @throws IllegalArgumentException
+     *             if selection contained item not available for selection
+     */
+    public void deselectItems(String... selection) {
+        ensureComponentIsUsable();
+        if (selection == null) {
+            getComponent().setValue(null);
+            return;
+        }
```

Comment:
Not sure about this one. If we deselect nothing, perhaps we should just leave value as is.
Or decide that null is not valid input for `deselectItems` and throw